packages/bot/src/scratch/blocks/Binary/Tick Analysis/tick_analysis.js

```diff
@@ -17,19 +18,22 @@ Blockly.Blocks.tick_analysis = {
                     check: null,
                 },
             ],
-            colour         : '#fef1cf',
-            colourSecondary: Blockly.Colours.Binary.colourSecondary,
-            colourTertiary : Blockly.Colours.Binary.colourTertiary,
+            colour         : Blockly.Colours.RootBlock.colour,
+            colourSecondary: Blockly.Colours.RootBlock.colourSecondary,
+            colourTertiary : Blockly.Colours.RootBlock.colourTertiary,
             tooltip        : translate('You can use this block to analyze the ticks, regardless of your trades'),
             category       : Blockly.Categories.Tick_Analysis,
         };
     },
     meta(){
         return {
-            'display_name': translate('Tick Analysis'),
-            'description' : translate('Tick Analysis Description'),
+            'display_name': translate('Run on every tick'),
+            'description' : translate('The content of this block is called on every tick. This blocks should be placed outside of any root block.'),
         };
     },
+    onchange() {
+        setBlockTextColor(this);
```

Comment:
Is this function still required?

Reply:
yes, it's used to change font color of root blocks after rendering